src/codemagic/apple/resources/enums.py

```diff
@@ -214,9 +218,9 @@ def is_tvos_profile(self) -> bool:
         return self.value.startswith('TVOS_')
 
     def devices_not_allowed(self) -> bool:
-        return not self.devices_allowed()
+        return not self.devices_required()
 
-    def devices_allowed(self) -> bool:
+    def devices_required(self) -> bool:
```

Comment:
This rename is a breaking change, but we need to persist backwords compatibility. As such I suggest you add an alias with deprectaion warning:
```python
    def devices_allowed(self) -> bool:
        # Deprecation warning saying that this method was renamed to
        # "devices_required" in version x.y.z and is subject to be removed
        # in future releases.
        log.get_logger(self.__class__).warning(...)
        return self.devices_required()
```